src/entries/popup/pages/settings/settings.tsx

```diff
@@ -199,43 +201,63 @@ export function Settings() {
               />
             }
           />
-        </Menu>
-        <Menu>
           <MenuItem
-            testId={'networks-link'}
             first
-            last
+            hasRightArrow
             leftComponent={
               <Symbol
-                symbol="network"
-                weight="medium"
+                symbol="bolt.fill"
+                color="red"
+                weight="semibold"
                 size={18}
-                color="green"
               />
             }
-            hasRightArrow
-            onClick={() => navigate(ROUTES.SETTINGS__NETWORKS)}
+            onClick={() => navigate(ROUTES.SETTINGS__TRANSACTIONS)}
             titleComponent={
-              <MenuItem.Title text={i18n.t('settings.networks.title')} />
+              <MenuItem.Title text={i18n.t('settings.transactions.title')} />
             }
+            testId="settings-transactions"
           />
+          {isWatchingWallet ? null : (
+            <MenuItem
+              first
```

Comment:
I think this need to have `last` here instead. I realized the tab-ing works, bit the highlight border radius are slightly wrong

Reply:
yeah i just updated thx